titapapunne/MoreThanWords
# #93: adding touchable opacity blocks for each focus area

- added an endpoint in app.py /calcProgressPodScreen that returns a dictionary with name, completed outcomes, and total outcomes 
- added touchable opacity component in FocusAreaBlock.js. Added a mapping function in TraineePod.js that outputs the component with respective data for each value in dictionary. 
- added navigation to each respective outcomes screen when click on the focus area (career expl., competency, life). there is still the bug with the react navigation lifecycles 

Steps to test PR: 
- Login in on app using Yichen or Punpun's account to test having real user data:
Username: my.waymao@gmail.com
password: ZijReFQ@AJYz3jfrA3iz

- Home screen should have trainee outcomes progress bar loaded. 
- Then after clicking on Trainee Pod from Home Screen, should be able to load all the progress bars with the correct number of total outcomes and completed outcomes.
- Then after clicking on any of the focus areas inside Trainee Pod, should be able to go to correct outcomes screen with correct tasks

Closes 
Standard Checklist:
- [ x ] Does the code work? Does it perform its intended function, the logic is correct etc.
- [ x ] Have you merged the most recent version of staging into your branch?
- [ x ] Are all merge conflicts resolved? 
- [ x ] Can any logging or debugging code be removed? (Any debugging statements should be removed)
- [ x ] Is the `diff` the **minimum amount** needed to achieve this ticket's goal?
